src/Config/Validator/Deploy/ElasticSearchUsage.php

```diff
@@ -56,16 +56,11 @@ public function __construct(
      */
     public function validate(): Validator\ResultInterface
```

Comment:
why do you need this separate class if  src/Config/Validator/Deploy/ElasticSearchVersion.php does the same validation + validation of versions

Reply:
Removed